frontend/src/queries/query.ts

```diff
@@ -93,6 +93,8 @@ export function queryExportContext<N extends DataNode = DataNode>(
     }
 }
 
+const SYNC_ONLY_QUERY_KINDS = ['HogQLMetadata', 'EventsQuery', 'HogQLAutocomplete'] satisfies NodeKind[keyof NodeKind][]
```

Comment:
Why `EventsQuery` here?

Reply:
That's a question for @webjunkie I believe

Reply:
It was.... you @mariusandra 👀 

https://github.com/PostHog/posthog/pull/18907